enso/experiment/__init__.py

```diff
@@ -327,11 +332,17 @@ def _split_dataset(dataset, training_size):
             yield splitter.split(np.zeros(len(dataset)), dataset[target]), target
 
     @staticmethod
-    def _load_dataset(dataset_name, featurizer_name):
+    def _load_dataset(dataset_name, featurizer_name, min_lf=0):
         """Responsible for loading a given dataset given the dataset_name and featurizer."""
         read_location = feature_set_location(dataset_name, featurizer_name)
         logging.info("Loading Dataset: %s" % read_location)
-        return joblib.load(read_location)
+        if min_lf == 0:
+            return joblib.load(read_location)
+        unfiltered = joblib.load(read_location)
+        label_counts = Counter([j[1] for j in unfiltered.Targets])
+        drop_labels = [i for i in label_counts if label_counts[i] >= min_lf]
```

Comment:
Confusing name for this variable

Reply:
Should also change to handle more than just rationalized classification before merge